grouper/background/background_processor.py

```diff
@@ -49,6 +50,9 @@ def _capture_exception(self):
         if self.sentry_client:
             self.sentry_client.captureException()
 
+    def crash(self):
+        os._exit(1)
```

Comment:
This seems aggressive. It will die in the middle of open transactions and not run any exception `finally` blocks or rollbacks. Does it not work to call `sys.exit(1)` like one normally would and let that exception propagate?

Reply:
In python, `sys.exit()` will raise will raise a `SystemExit` exception, which will cause this thread to exit. However, the process will not terminate unless `sys.exit()` is called from the main thread. The only other implementation I can think of is something causing the main thread to exit, which will be tricky because the main thread will usually be in an endless `app.run()` call or something.

https://stackoverflow.com/questions/905189/why-does-sys-exit-not-exit-when-called-inside-a-thread-in-python

Reply:
Oh, ugh. Yeah, in that case `_exit` is probably the best we can do.